addons/calendar/static/src/js/base_calendar.js

```diff
@@ -122,7 +122,18 @@ BasicModel.include({
      */
     _fetchSpecialAttendeeStatus: function (record, fieldName) {
         var context = record.getContext({fieldName: fieldName});
-        var attendeeIDs = record.data[fieldName] ? this.localData[record.data[fieldName]].res_ids : [];
+        let attendeeIDs = [];
+        if (record.data[fieldName]) {
+            // Note: this widget used in calendar popover and calendar popover
+            // use makeRecord which creates relational datapoint so we already
+            // have res_ids in record.data[fieldName]
+            if (_.isObject(record.data[fieldName])) {
+                attendeeIDs = record.data[fieldName].res_ids;
+            } else {
```

Comment:
could you just write `const attendeeIDs = record.data[fieldName] ? record.data[fieldName].res_ids : [];?

Reply:
👍

Reply:
ok so let us go back to previous version here:
attendeeIDs = record.data[fieldName].res_ids